xc7/primitives/slicem/Ndram/ntemplate.N_dram.pb_type.xml

```diff
@@ -18,11 +18,11 @@
       <input  name="in"  num_pins="5" port_class="lut_in" />
       <output name="out" num_pins="1" port_class="lut_out" />
       <delay_matrix type="max" in_port="in" out_port="out">
-        0.068e-9
-        0.068e-9
-        0.068e-9
-        0.068e-9
-        0.068e-9
+        {{iopath_A1_O5}}
```

Comment:
Why is this double escaped?

Reply:
because it first goes through templating (`n.py`) which also uses python's `.format`. The same happens with LUTs in Nlut.pb_type.xml